src/main/java/org/jabref/logic/importer/fileformat/PdfMergeMetadataImporter.java

```diff
@@ -65,6 +67,12 @@ public PdfMergeMetadataImporter(ImportFormatPreferences importFormatPreferences)
         if (importFormatPreferences.grobidPreferences().isGrobidEnabled()) {
             this.metadataImporters.add(2, new PdfGrobidImporter(importFormatPreferences));
         }
+        doiFetcher = new DoiFetcher(importFormatPreferences);
+        arXivFetcher = new ArXivFetcher(importFormatPreferences);
+
+        isbnFetcher = new IsbnFetcher(importFormatPreferences);
+        // .addRetryFetcher(new EbookDeIsbnFetcher(importFormatPreferences))
```

Comment:
why is this commented out?

Reply:
It already was commented out...

- 2023: 3d136614 / https://github.com/JabRef/jabref/pull/10146 - Please ask @Siedlerchr for this thoughts two years ago ^^
- 2022: #9465